modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/asset/model/DepotAssetRendererFactoryWrapper.java

```diff
@@ -111,6 +114,15 @@ public AssetRenderer<T> getAssetRenderer(long classPK)
 			return assetRenderer;
 		}
 
+		DepotEntry depotEntry = _depotEntryLocalService.getGroupDepotEntry(
+			groupId);
+
+		if (FeatureFlagManagerUtil.isEnabled("LPD-17564") &&
```

Comment:
Maybe we could separate this in 2 and do an early null return if FeatureFlagManagerUtil.isEnabled("LPD-17564") is not enabled to avoid call depotLocalService in this case

Reply:
also, shouldn't we have to add the companyId to the isEnabled method to avoid errors in runtime?